catalog/tests/dags/providers/provider_api_scripts/test_wordpress.py

```diff
@@ -61,11 +61,11 @@ def test_get_should_continue_checks_total_pages(
 
 
 @pytest.mark.parametrize("missing_field", ["slug", "link"])
-def test_get_record_data_returns_none_when_missing_necessary_data(
+def test_get_record_data_returns_none_when_necessary_data_is_falsy(
     ingester, missing_field
 ):
     image_data = _get_resource_json("full_item.json")
-    image_data.pop(missing_field, None)
+    image_data[missing_field] = ""
```

Comment:
Could we add a check for when the field is missing, in addition to when it is falsy?